### PR TITLE
Create schemas

### PR DESCRIPTION
**Pull request by Serrones**
Create schemas, adjust models and update migration

### CODE REVIEW

File: migrations/versions/f6598fcf89f5_.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-"""initial migration
+"""empty message
```
Comment (jfunez) [inline migrations/versions/f6598fcf89f5_.py]: I believe that "initial migration" is better than "empty message"

File: sucuri/models/models.py [hunk 2]
```diff
@@ -102,6 +89,4 @@ class Label(Base):
     id = Column(Integer, primary_key=True, index=True)
```
Comment (jfunez) [inline sucuri/models/models.py:90]: move the `id` as the first attribute of the class

**Review by jfunez — CHANGES_REQUESTED** [review]

File: sucuri/crud/crud_profile.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+
+def get_profiles(skip: int = 0, limit: int = 100, db: Session = Depends(get_db)):
+    return db.query(Profile).offset(skip).limit(limit).all()
+
+
+def delete_profile(profile_id: int, db: Session = Depends(get_db)):
+    profile = db.query(Profile).filter(Profile.id == profile_id).first()
+    if profile is None:
+        raise HTTPException(status_code=404, detail="Profile not found")
```
Comment (jfunez) [inline sucuri/crud/crud_profile.py]: replace with another exception

**Review by jfunez — CHANGES_REQUESTED** [review]

File: sucuri/crud/crud_profile.py (hunk 3 above)
Comment (Serrones) [reply sucuri/crud/crud_profile.py]: This code was removed

**Review by jfunez — APPROVED** [review]

**Review by jlugao — APPROVED** [review]

**Pull request merged**